source/documentation/tools/create-a-derived-table/database-access.md

```diff
@@ -1,5 +1,10 @@
 # Database Access
 
+To get access to AP data, users must use the [data-engineering-database-access](https://github.com/moj-analytical-services/data-engineering-database-access) service. [Database resources](https://github.com/moj-analytical-services/data-engineering-database-access/tree/main/database_access) define access for databases, and these are assigned to users via [project files](https://github.com/moj-analytical-services/data-engineering-database-access/tree/main/project_access).
```

Comment:
Think it needs to assume the user has nothing setup. So a step before the first here would be something like:
- Have a github account and be part of the `moj-analytical-services` github organisation. [Guidance for joining github organisations](https://github.com/orgs/moj-analytical-services/sso)

Reply:
I linked the AP quickstart guide which has this info